server/src/database/entities/Image.ts

```diff
@@ -13,4 +21,20 @@ export class Image extends BaseEntity {
   })
   @IsString()
   public filePath!: string;
+
+  get fileName(): string {
+    const [ fname ] = this.filePath.split('/').reverse()
+
+    return fname
+  }
+
+  get src(): string {
+    return (
+      `http://localhost:${process.env.NODE_PORT}/images/${this.fileName}`
```

Comment:
we should probably use `NODE_HOST` instead of `localhost`

Reply:
yeah, probably abstract the getUrl and upi could probably decide how to pass the API url to the FE, let's leave it at this in this PR, since you also have Axios URL statically set in FE. We would tackle this during next walk.